package.json

```diff
@@ -18,7 +18,8 @@
     "webpack-dev-server": "^1.9.0"
   },
   "scripts": {
-    "start": "webpack-dev-server --progress --colors",
+    "start": "webpack-dev-server --progress --colors --port 9000",
+    "sourcemap": "webpack --dev-tool source-map",
     "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
I'm not sure if 'sourcemap' is a development artifact and should be removed, or if it's meant to save both the source map and the main bundle to the local file system -- if that's the case, maybe a more general name the deemphasizes the source-map ('build'?), and changing it to `webpack --config webpack.config.js`.